metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -134,28 +134,36 @@ final class BloopServers(
       maybeBloopJvmProperties: Option[List[String]],
       maybeJavaHome: Option[String]
   ): Try[Unit] = Try {
+    if (maybeJavaHome.isDefined || maybeBloopJvmProperties.isDefined) {
+      val jvmPropertiesString =
+        (maybeBloopJvmProperties, maybeJavaHome) match {
+          case (None, Some(javaHome)) =>
+            ujson.write(
+              ujson.Obj(
+                "javaHome" -> javaHome
+              )
+            )
+          case (Some(javaOptions), Some(javaHome)) =>
+            ujson.write(
+              ujson.Obj(
+                "javaHome" -> javaHome,
+                "javaOptions" -> javaOptions.map(ujson.Str(_))
+              )
+            )
+          case (Some(javaOptions), None) =>
+            ujson.write(
+              ujson.Obj(
+                "javaOptions" -> javaOptions.map(ujson.Str(_))
+              )
+            )
+          case _ => ""
```

Comment:
Is it ok to leave it empty?
it looks like like previously the empty was written as an object `{}`

Reply:
This can't really happen since we don't write anything if both are None.